doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -1290,18 +1308,44 @@ PSI Variables by Alpha
    The total electronic energy [Eh] and correlation energy component [Eh]
    for the orbital-optimized MP2 level of theory.
 
+.. psivar:: OMP2 REFERENCE CORRECTION ENERGY
+
+   The additional correction to the SCF reference energy [Eh]
```

Comment:
It's unclear to me what an "additional correction to the SCF reference energy" is, but I would have guessed "a correlation energy."

Please change **all** descriptions of variables named like "X REFERENCE CORRECTION ENERGY" to something along the lines of:
`The difference between the single-determinant energy [Eh] of the final and initial orbitals for the X level of theory.`